.github/workflows/release.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Creating .npmrc
         run: |
           cat << EOF > "$HOME/.npmrc"
+            provenance=true
```

Comment:
This is based on the documentation: https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools